data/make-bundle-dir.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/sh
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2016 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+
+# Description: Script to create a bundle directory that can be made
+#   into a full OCI bundle when a "config.json" file is created within it.
+#
+# On success, this script will create the directory specified as a
+# bundle directory. This directory can be used as an OCI bundle once it
+# has had a "config.json" file added to it. The directory created will
+# contain a sub-directory called "rootfs" (which is the canonical path to
+# the rootfs specified by an OCI bundle).
+#
+# See the following for further details:
+#
+# - https://github.com/opencontainers/runtime-spec/blob/master/bundle.md
+
+# Name of a small docker image that create_rootfs() uses
+# to make a chroot directory.
+docker_image="busybox"
```

Comment:
@devimc I think the intention here is to make this test as fast as possible and busybox image is small enough